src/components/books/list.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-unused-vars */
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { addBook } from '../../redux/books/books';
+import Books from './books.js';
+import Form from './form.js';
+import './list.css';
+
+function List() {
+  const bookList = useSelector((state) => state.booksReducer);
+  console.log(bookList);
```

Comment:
**Optional Changes**
Your `book` component  is nicely done. Bravo! However;
- [ ] It's good practice to allows delete `console.log` codes before pushing into `git/version` control. Let as remove these. Many devs have different opinions, but at least most of them agree that you shouldn't push to version control.